polkadot/Cargo.toml

```diff
@@ -30,6 +30,7 @@ polkadot-cli = { path = "cli", features = [ "westend-native", "rococo-native" ]
 polkadot-node-core-pvf = { path = "node/core/pvf" }
 polkadot-node-core-pvf-prepare-worker = { path = "node/core/pvf/prepare-worker" }
 polkadot-overseer = { path = "node/overseer" }
+tracking-allocator = { path = "node/tracking-allocator" }
```

Comment:
I think this can be removed also? At least the comment above no longer applies here:

> Crates in our workspace, defined as dependencies so we can pass them feature flags.